Cargo.toml

```diff
@@ -35,7 +36,7 @@ picky-asn1-der = "0.4"
 picky-asn1-x509 = "0.12"
 predicates = { version = "3.1.3" }
 pretty_env_logger = "0.5"
-reqwest = {version = "0.12", default-features = false, features = ["json", "default-tls"]}
+reqwest = {version = "0.12", default-features = false, features = ["json", "native-tls"]}
```

Comment:
Good point. I was under the impression we had to use native-tls explictly instead of default-tls, since we have the goal to use openssl on Linux, as default-tls might change its backend in the future.